### PR TITLE
[4.0] RavenDB-9624

### PR DESCRIPTION
**Pull request by aviv86**

### CODE REVIEW

File: src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs [hunk 1]
```diff
@@ -61,14 +61,14 @@ internal class RavenQueryProviderProcessor<T>
             "AS",
             "SELECT",
             "WHERE",
-            "LOAD",
```
Comment (ayende) [inline src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs:64]: why did you remove it?

Comment (aviv86) [reply src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs:64]: because load is also reserved in js
https://github.com/ravendb/ravendb/blob/71323ce78bd7d1b043341b51b73c09db43e37927/test/SlowTests/Issues/RavenDB-9624.cs#L131

Comment (aviv86) [reply src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs:64]: but now i realize that this query won't work because of it:
from o in session.Query<Order>() select new { Load = o.Company };
